app/views/static_pages/home.html.erb

```diff
@@ -1,7 +1,6 @@
-<% provide :title, t(:home) %>
 <h1><%= t :home %></h1>
 <p>
-  This is the home page for the
-  <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
-  sample application.
+  <%= t :f_title %>
+  <a href="http://www.railstutorial.org/"><%= t :RoR %></a>
+  <%= t :l_title %>
 </p>
```

Comment:
Dài dòng quá em. Chỉ cần 1 mesage thôi, sao phải tách làm 3 message ri.
Ví dụ:
t(".content_p", links: link_to(t(".a_href"), "https://www.railstutorial.org"))

Và trong file en, em chỉ cần định nghĩa message:
home:
    content_p: "This is the home page for the %{links} sample application."
    a_href: Ruby on Rails Tutorial

Reply:
Dạ, để em sửa ạ. Em mới học mà :)